docs/core/whats-new/dotnet-8.md

```diff
@@ -1093,21 +1152,21 @@ This section contains the following subtopics:
 
 MSBuild includes a new feature that makes it easier to incorporate data from MSBuild into your scripts or tools. The following new flags are available for CLI commands such as [dotnet publish](../tools/dotnet-publish.md) to obtain data for use in CI pipelines and elsewhere.
 
-| Flag  | Description  |
-|---------|---------|
-| `--getProperty:<PROPERTYNAME>` | Retrieves the MSBuild property with the specified name. |
-| `--getItem:<ITEMTYPE>` | Retrieves MSBuild items of the specified type. |
+| Flag                              | Description                                              |
+|-----------------------------------|----------------------------------------------------------|
+| `--getProperty:<PROPERTYNAME>`    | Retrieves the MSBuild property with the specified name.  |
+| `--getItem:<ITEMTYPE>`            | Retrieves MSBuild items of the specified type.           |
 | `--getTargetResults:<TARGETNAME>` | Retrieves the outputs from running the specified target. |
 
 Values are written to the standard output. Multiple or complex values are output as JSON, as shown in the following examples.
 
 ```dotnetcli
->dotnet publish --getProperty:OutputPath
+dotnet publish --getProperty:OutputPath
```

Comment:
@IEvangelist I think if you remove the caret here (and below in a few places), you need to separate the output into a different code block, otherwise it all just looks like a command.

Reply:
Ah, ok... I saw the `dontetcli` slug and assumed these were commands. So you're using `>` to indicate output?